UnitTestFrameworkPkg/Framework.md

```diff
@@ -0,0 +1,306 @@
+// TODO - should there be an intro blurb linking back to the main ReadMe?

```

Comment:
TODO: Fix TODO and resolve comment.

To your question, I think that is a good idea.